src/adapters/refract/getUriParameters.coffee

```diff
@@ -1,29 +1,25 @@
 lodash = require('./helper')
+minim = require('./minim')
 getMetaDescription = require('./getMetaDescription')
 
 getUriParameters = (hrefVariables, options) ->
-  hrefVariablesContent = lodash.content(hrefVariables)
-
-  if hrefVariablesContent is undefined
+  if hrefVariables is undefined
     return undefined
 
-  hrefVariablesContent.map((hrefVariable) ->
-    lodashedHrefVariable = lodash.chain(hrefVariable)
-    typeAttributes = lodashedHrefVariable
-                        .get('attributes.typeAttributes')
-                        .value()
+  hrefVariables.content.map((hrefVariable) ->
+    typeAttributes = hrefVariable.attributes.get('typeAttributes')?.toValue()
 
     if typeAttributes
       required = typeAttributes.indexOf('required') isnt -1
```

Comment:
I think this can be simplified and prevent the need to `toValue()` the type attributes above.

```cs
typeAttributes = hrefVariable.attributes.get('typeAttributes')
required = typeAttributes?.contains('required') || false
```